apps/ledger-live-mobile/src/components/withRemountableWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useCallback, useState } from "react";
+
+/**
+ * Use this to wrap your component and pass it a function "remountMe" as a prop
+ * that simply remounts the component.
+ * */
+export default function withRemountableWrapper<T>(
+  Component: React.ComponentType<T & { remountMe: () => void }>,
```

Comment:
what do you think of the name `rerenderMe` ? "remount" could be misleading i feel

Reply:
it's a remount with a new instance of the component, not a rerender